src/pages/singlePagePost.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <div class=" m-auto  ">
+        <div class="text-right p-4">
+            <v-btn class="mt-2 pointer mr-2" color="blue" @click="goback()">
+                Go Back
+            </v-btn>
+        </div>
+        <div v-if="post.title" class=" w-4/6 m-auto bg-gray-300 border border-gray-300 rounded-xl">
+            <h2 class=" text-left font-bold p-4 text-xl ml-15 mr-15">{{ post.title }}</h2>
+            <div class=" p-4 flex items-center h-10 mt-1 ml-15">
+                <img :src="post.profilePhotoPath" alt="Profile Photo" class=" w-13 h-12 rounded-full mr-4" />
+                <span>
+                    <h2 class=" text-sm font-bold">{{ post.firstName }} {{ post.lastName }}</h2>
+                    <h4 class="text-xs ">{{ new Date(post.updatedAt).toLocaleDateString() }}
+                        {{ new Date(post.updatedAt).toLocaleTimeString() }}
+                    </h4>
+                </span>
+            </div>
+            <img :src="post.photo" alt="Post Photo" class=" w-full  m-auto  mt-5 object-cover" />
+            <p class=" ml-15 mr-15 text-xs mt-10 " v-html="post.description"></p>
+            <div class="mb-3 mt-6 ml-15">
+                <span v-for="tagUser in post.taggedUsers" :key="tagUser.id" chips class="text-left ml-3  font-light ">
+                    <span class="border border-gray-400 p-2 rounded-md bg-black text-white ">{{ tagUser }}</span>
+                </span>
+            </div>
+            <div class="p-3 ml-15">
+                <div class=" w-4/5">
+                    <div v-for="commentDetails in allcomments" :key="commentDetails.id"
```

Comment:
this is duplicate, can you create a common component for this and same for user tagged list and use it both pages
list and single post @ShivanshSinghnnovative